crates/turbopack-ecmascript/src/side_effect_optimization/locals/module.rs

```diff
@@ -54,9 +55,9 @@ impl Module for EcmascriptModuleLocalsModule {
 impl Asset for EcmascriptModuleLocalsModule {
     #[turbo_tasks::function]
     fn content(&self) -> Vc<AssetContent> {
-        // This is not reachable because EcmascriptModuleLocalsModule implements
-        // ChunkableModule and ChunkableModule::as_chunk_item is called instead.
-        todo!("EcmascriptModuleLocalsModule::content is not implemented")
+        let f = File::from("");
+
+        AssetContent::file(FileContent::Content(f).cell())
```

Comment:
This should return the original module content

```suggestion
        self.module.content()
```